### DISCUSSION TITLE
Output RHS or RHS Jacobian

### DISCUSSION THREAD

**Discussion by aph4499** [usnistgov/fipy#1040] [Q&A]
Hello,
Is there a way to output the RHS Jacobian of a system of equations in Fipy or simply the RHS?
Thanks

**Comment by guyer** [comment]
The [`FIPY_DISPLAY_MATRIX`](https://pages.nist.gov/fipy/en/latest/USAGE.html#envvar-FIPY_DISPLAY_MATRIX) environment variable allows you to see the matrix and RHS.

Alternatively, call `eq.cacheRHSvector()` before solving or sweeping and then access `eq.RHSvector` after.

**Reply by aph4499** [reply]
Thanks
